Gems/ROS2/Code/Source/RobotImporter/URDF/UrdfParser.h

```diff
@@ -10,16 +10,17 @@
 
 #include <AzCore/IO/Path/Path.h>
 #include <AzCore/std/utility/expected.h>
-#include <sdf/Root.hh>
+#include <SdfAssetBuilder/SdfAssetBuilderSettings.h>
```

Comment:
Actually this include isn't needed in the UrdfParser.h.

The SdfAssetBuilderSettings can be forward declared in this header and then included in the cpp file.
That will help a little bit with compile times.
```suggestion
```